x-pack/plugin/security/build.gradle

```diff
@@ -476,6 +484,11 @@ test {
   systemProperty 'es.transport.buffer.size', '256k'
 }
 
+internalClusterTest {
```

Comment:
I suspect this probably isn't needed for unit tests. Can we ditch the block above?

Reply:
yup: https://github.com/elastic/elasticsearch/pull/61802/commits/1f4ca9a0ea0276a9c23b9ec69b4ad0e1f6a01bf7